### ISSUE TITLE
migrate fails on documents.0025_auto_20150718_0742

### ISSUE THREAD

**Issue by simeon-walker** [mayan-edms/mayan-edms#250]
I am running migrate in Mayan v.2.1.10 against a 1.1.0 database in Oracle:
```
Applying documents.0025_auto_20150718_0742...Traceback (most recent call last):
  File "manage.py", line 9, in <module>
    execute_from_command_line(sys.argv)
  File "/home/vagrant/Projects/edms_project/trunk/virtenv/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/home/vagrant/Projects/edms_project/trunk/virtenv/lib/python2.7/site-packages/django/core/management/__init__.py", line 346, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/vagrant/Projects/edms_project/trunk/virtenv/lib/python2.7/site-packages/django/core/management/base.py", line 394, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/vagrant/Projects/edms_project/trunk/virtenv/lib/python2.7/site-packages/django/core/management/base.py", line 445, in execute
    output = self.handle(*args, **options)
  File "/home/vagrant/Projects/edms_project/trunk/virtenv/lib/python2.7/site-packages/django/core/management/commands/migrate.py", line 222, in handle
    executor.migrate(targets, plan, fake=fake, fake_initial=fake_initial)
  File "/home/vagrant/Projects/edms_project/trunk/virtenv/lib/python2.7/site-packages/django/db/migrations/executor.py", line 110, in migrate
    self.apply_migration(states[migration], migration, fake=fake, fake_initial=fake_initial)
  File "/home/vagrant/Projects/edms_project/trunk/virtenv/lib/python2.7/site-packages/django/db/migrations/executor.py", line 148, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/home/vagrant/Projects/edms_project/trunk/virtenv/lib/python2.7/site-packages/django/db/migrations/migration.py", line 112, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/home/vagrant/Projects/edms_project/trunk/virtenv/lib/python2.7/site-packages/django/db/migrations/operations/special.py", line 183, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/home/vagrant/Projects/edms_project/trunk/virtenv/lib/python2.7/site-packages/mayan/apps/documents/migrations/0025_auto_20150718_0742.py", line 13, in change_bibliographic_to_terminology
    language = pycountry.languages.get(bibliographic=document.language)
  File "/home/vagrant/Projects/edms_project/trunk/virtenv/lib/python2.7/site-packages/pycountry/db.py", line 21, in load_if_needed
    return f(self, *args, **kw)
  File "/home/vagrant/Projects/edms_project/trunk/virtenv/lib/python2.7/site-packages/pycountry/db.py", line 128, in get
    return self.indices[field][value]
KeyError: 'bibliographic'
```
I have been unable to find a version of pycountry that enables this to work.

**Comment by rosarior** [comment]
That was a data migration to move document languages from bibliographic` to `terminology` (https://github.com/mayan-edms/mayan-edms/issues/211). Seems new version of PyCountry just removed the bibliographics key. 

The ideal solution would be to find a way to have two PyCountry versions installed at the same time, which I don't think is possible.

Since this is a data migration there should not be any adverse effects to the schema from adding a try: except: block to will allow it to run even with modern PyCountry packages. The only effect is that the language of existing documents will need to be refreshed either from the UI or with a script.

**Comment by rosarior** [comment]
Fix in 3d6a32c7b6c9cf8f2f5eb4686143ef76f31f717a. Thanks!
